octobot_trading/exchanges/traders/trader.py

```diff
@@ -105,7 +105,8 @@ async def create_order(self, order, loaded: bool = False, pre_init_callback=None
         if pre_init_callback is not None:
             await pre_init_callback(new_order)
 
-        await new_order.initialize()
+        # force initialize to always create open state
+        await new_order.initialize(force=True)
```

Comment:
I don't understand this `force=True` there is no force usage in `create_order_state`

Reply:
right

Reply:
actually force was to force initialization even if already initialized. However as it's unclear, I changed directly the `is_initialized attribute`